src/sparse.js

```diff
@@ -141,11 +141,11 @@ async function populate(chunks, blockSize) {
     const data = chunk.data;
 
     if (chunkType === ChunkType.Raw) {
-      const rawData = new Uint8Array(data.arrayBuffer());
+      const rawData = new Uint8Array(await data.arrayBuffer());
```

Comment:
Not sure how this works currently...

Reply:
was refactored without await in https://github.com/commaai/qdl.js/pull/38

https://github.com/commaai/qdl.js/pull/38/files#diff-ac49acb9d30956c6cfb54b0868eeed5878024313b70a54404dada946c1d2cb8cR144

Reply:
oh good that isn't shipped yet

thanks!